src/includes/performance-monitoring/configuration/javascript.mdx

```diff
@@ -130,23 +131,33 @@ Sentry.init({
 **Manual Instrumentation**
 
 To manually instrument certain regions of your code, you can create a transaction to capture them.
-This is valid for both JavaScript Browser and Node and works independently of the `Tracing` integration.
+This is valid for both JavaScript Browser and Node and works independently of the `BrowserTracing` integration.
 
 ```javascript
-const transaction = Sentry.startTransaction({name: 'test-transaction'});
-const span = transaction.startChild({op: 'functionX'}); // This function returns a Span
+const transaction = Sentry.startTransaction({ name: "test-transaction" });
+const span = transaction.startChild({ op: "functionX" }); // This function returns a Span
 // functionCallX
 span.finish(); // Remember that only finished spans will be sent with the transaction
 transaction.finish(); // Finishing the transaction will send it to Sentry
 ```
 
+If you want the transaction to be associated with any global errors, or have integrations automatically add spans to your transaction, the transaction should be set on the scope.
+
+```javascript
+const transaction = Sentry.startTransaction({ name: "test-transaction" });
+
+Sentry.configureScope(scope => scope.setSpan(transaction));
```

Comment:
This section is new - added as I figured this would be a common concern to bring up.

Reply:
Hmm, I don't like sneaking this in "Migrate to @sentry/tracing" PR.

We could as well decide that startTransaction always updates the scope, making it less of a burden to associate errors and transactions.